parsedown

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env php
+<?php
+
+require(dirname(__FILE__).'/Parsedown.php');
+
+// From a file
+if (isset($argv[1])) {
+
+	if ( ! is_readable($argv[1])) {
+		throw new RuntimeException(sprintf(
+			'Could not read "%s"',
+			$argv[1]
+		));
```

Comment:
how does the exception appear to the user of the script? I would prefer a more userfriendly echo + exit(1) here.